tests/modules/notebooks/test_sagemaker_notebook.py

```diff
@@ -10,75 +20,35 @@ def org1(org, user, group, tenant):
 
 
 @pytest.fixture(scope='module')
-def env1(env, org1, user, group, tenant, module_mocker):
+def env1(env, org1, user, group, tenant, db, module_mocker):
     module_mocker.patch('requests.post', return_value=True)
     module_mocker.patch(
         'dataall.api.Objects.Environment.resolvers.check_environment', return_value=True
     )
-    env1 = env(org1, 'dev', user.userName, group.name, '111111111111', 'eu-west-1')
+    env1 = env(org1, 'dev', user.userName, group.name, '111111111111', 'eu-west-1',
+               parameters={"notebooksEnabled": "True"})
     yield env1
 
 
-@pytest.fixture(scope='module', autouse=True)
```

Comment:
Moved to `tests/modules/notebooks/conftest.py` right?

Reply:
Yeah! Since it's related only to notebooks I decided to move tests related to notebooks to `modules` folder